### PR TITLE
Test

### PR DESCRIPTION
**Pull request by Falchy**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by amrsoll** [comment]
You should probably go through the checklist on your Pull Request (PR)

File: keyboards/manuform/info.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["C2", "C2", "C2", "C2", "C2", "C2", "C2", "C2", "C2", "C2", "C2", "C2", "C2", "C2"],
+        "rows": ["D1", "D1", "D1", "D1"]
```
Comment (drashna) [inline keyboards/manuform/info.json:17]: These are definitely wrong.

File: keyboards/manuform/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/manuform/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

**Review by drashna — COMMENTED** [review]
Also, this looks to be a dactyl manuform.  If so, there is already a folder for it, but not for the 3x6, located in /keyboards/handwired/dactyl_manuform

**Pull request closed without merging**